app/controllers/media_objects_controller.rb

```diff
@@ -237,6 +237,7 @@ def update_media_object
         master_file.title = file_spec[:label] if file_spec[:label].present?
         master_file.date_digitized = DateTime.parse(file_spec[:date_digitized]).to_time.utc.iso8601 if file_spec[:date_digitized].present?
         master_file.identifier += Array(file_spec[:other_identifier])
+        master_file.comment += Array(params[:files][index][:comment])
```

Comment:
Is the comment not in `file_spec` or is this just to workaround the strong params issue?

Reply:
Correct. Adding the comment to the strong parameters worked, in so far as the comment parameter wasn't rejected. However the values weren't there in the permitted parameter.